.devcontainer/contributor/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Radius - Contributor",
+    "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+    "features": {
+        "ghcr.io/devcontainers/features/go:1": {},
+        "ghcr.io/devcontainers/features/git:1": {},
+        "ghcr.io/devcontainers/features/github-cli:1": {},
+        "ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {
+            "minikube": "none"
+        },
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {
+            "version": "latest",
+            "moby": true
+        }
```

Comment:
Just for my own understanding, what do the numbers represent here?

Reply:
The numbers are the (major) versions of the *features* as defined here <https://containers.dev/implementors/features/#devcontainer-json-properties>.

The version of the content of the features is usually specified via the options as value to the `version` key.